docs/basic-features/eslint.md

```diff
@@ -161,7 +149,19 @@ ESLint also contains code formatting rules, which can conflict with your existin
 
 ## Migrating Existing Config
 
-If you already have ESLint configured in your application, we recommend extending directly from the Next.js ESLint plugin instead of the shareable configuration.
+### Recommended Plugin Ruleset
+
+If you already have ESLint configured in your application and any of the following conditions are true:
+
+- You have one or more of the following plugins already installed (either separately or through a different config such as `airbnb` or `react-app`):
+  - `react`
+  - `react-hooks`
+  - `jsx-a11y`
+  - `import`
+- You've defined specific `parserOptions` that need to be different how Babel is configured within Next.js
```

Comment:
```suggestion
- You've defined specific `parserOptions` that are different from how Babel is configured within Next.js
```

And how it this? Can we link to something?

Reply:
Updated to include a little more detail and a link:

```diff
- You've defined specific `parserOptions` that are different from how Babel is configured within Next.js
+ You've defined specific `parserOptions` that are different from how Babel is configured within Next.js (this is not recommended unless you have [customized your Babel configuration](https://nextjs.org/docs/advanced-features/customizing-babel-config))
```